app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@chessbyte Doesn't look like that begin/rescue block in ci_processing/process_elements works as intended and is able to trap failures in models. CBR controller, for example, sets its own flash messages to deal with that. New delete logic (above) now uses messages generated in model/ci_processing to display in UI, but when adding/saving/deleting rates all messages for UI are generated by the controller itself.